src/components/AddBook.js

```diff
@@ -1,12 +1,27 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBooktolist } from '../redux/books/books';
 
-const AddBook = () => (
-  <div className="entry">
-    <h1>Add New Book</h1>
-    <input type="text" placeholder="Enter title of the book" id="title" />
-    <input type="text" placeholder="Enter author name" id="author" />
-    <button type="button" text="Add" value="Add" label="Add">Add Book</button>
-  </div>
-);
+const AddBook = () => {
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log('Form submitted');
```

Comment:
- [ ] Kindly remove console.log from your code to make it clean.

Reply:
@omarsalem7 Thank you for the feedback. I will remove this statement now.